src/main/java/com/revature/backend/model/Associate.java

```diff
@@ -0,0 +1,190 @@
+package com.revature.backend.model;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "associate")
+public class Associate {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+
+    @Column(name = "salesforce_id")
+    private String salesforce_id;
```

Comment:
Please fix all the fields, getters, and setters in this class to also be camelCase.

Reply:
I agree SWOT__ID does look strange, SwotID?

Reply:
I believe I got all of them.